composer.json

```diff
@@ -20,20 +20,22 @@
             "homepage": "https://github.com/friendsofsymfony/FOSHttpCache/contributors"
         }
     ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
-        "php": ">=5.4.8",
+        "php": ">=5.5.9",
         "symfony/event-dispatcher": "^2.3||^3.0",
         "symfony/options-resolver": "^2.3||^3.0",
-        "psr/http-message-implementation": "~1.0",
-        "php-http/adapter-implementation": "^0.1.0",
-        "php-http/discovery": "^0.1.1",
-        "php-http/message-decorator": "^0.1.0"
+        "php-http/client-implementation": "^1.0.0",
+        "php-http/discovery": "^0.6.0",
+        "php-http/message": "^0.2.1",
+        "puli/cli": "^1.0.0-beta8"
```

Comment:
>  But I guess some time in the future having puli.phar in the users prefix will be as common as composer.

True, but we need to be as user/dev-friendly as possible in the meantime. :wink:

The best solution now is for us to make Discovery an optional dependency. Installing that would be easiest if Discovery required Puli.